robocopkaka/bootcamp-29-project
# Feat create edit center components

#### What does this PR do?
Create a component for editing centers
#### Description of Task to be completed?
* Write tests for component
* Create component
* Use componentWillReceive props to pass a center's properties to the component's internal state
#### How should this be manually tested?
+ Go to the centers page
+ Click on the edit icon on any center
+ Modify any detail and click on `Update Center`
+ The center should be updated with the new details
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?(if applicable)
#155167153
